cmd/res/profiles/generic.yaml

```
@@ -7,25 +7,57 @@ description: "Example of Device-Uart"
 
 deviceResources:
 -
-    name: "Read_UART"
+    name: "Read_String"
```

Resources should not be named with Read/Write in name. That is what readWrite propertie is for.
```suggestion
    name: "String"
```

---

Addressed.